### ISSUE TITLE
Crysis Warhead crashes on launch

### ISSUE THREAD

**Issue by vlad54rus** [ccomrade/c1-launcher#39]
Here's the log file: 
[Game.log](https://github.com/ccomrade/c1-launcher/files/14312441/Game.log)

**Comment by ccomrade** [comment]
Ahhh it crashes because it fails to load Logitech G15 display resources from CryGame (Warhead EXE). The problem is that `g_hInst` is null. It's not initialized because `WinMain` is not executed (Warhead EXE is loaded as a "DLL").

I'm amazed that you managed to find this.

**Comment by ccomrade** [comment]
Fix implemented. Could you test it? I don't have Logitech G15 here unfortunately. A rare hardware these days.

[c1-launcher-v6-build-warhead.zip](https://github.com/ccomrade/c1-launcher/files/14324530/c1-launcher-v6-build-warhead.zip)

**Comment by ccomrade** [comment]
Interesting. Anyway, thanks for testing and reporting the issue. I will close this once v7 is released, so people having the same problem can find these fixed EXEs in the meantime.

**Comment by vlad54rus** [comment]
I think the changelog should be corrected. The crash happens when Logitech Gaming Software in installed on the user's system, regardless to the actual hardware connected.

**Comment by ccomrade** [comment]
You're right. Fixed in https://github.com/ccomrade/c1-launcher/commit/7f255e9f661d00eacccd583d4554a020ebe76370. Thanks.

**Comment by ccomrade** [comment]
Fixed in [v7](https://github.com/ccomrade/c1-launcher/releases/tag/v7). Closing.
